docs/swagger/spec.yml

```diff
@@ -783,6 +818,53 @@ paths:
       responses:
         204:
           description: Successfully updated order of scenes in project
+
+  /aoi/{uuid}/:
```

Comment:
I considered that initially. Would that endpoint be useful given `/projects/{uuid}/aoi/` already exists?